src/main/java/com/marklogic/kafka/connect/source/AbstractRowBatchBuilder.java

```diff
@@ -16,11 +16,14 @@ public class AbstractRowBatchBuilder extends LoggingObject {
     protected final DataMovementManager dataMovementManager;
     protected final Map<String, Object> parsedConfig;
     protected final String topic;
+    protected final String constrainingColumn;
```

Comment:
These two should be private since no one needs them (yet) except for this class.